tests/test_helpers.py

```diff
@@ -72,6 +72,11 @@ def test_local_model_non_default_name(self, tmp_path):
         assert result is True
 
 
+@pytest.fixture(scope="class")
```

Comment:
Please move this inside of the `TestScalingAdapters` class. Don't forget to add the `self` argument to the signature.